sources/Util/Utils.hpp

```diff
@@ -0,0 +1,26 @@
+//
+// Created by cbihan on 26/05/2021.
+//
+
+#pragma once
+
+#include "Vector/Vector3.hpp"
+#include "Models/Vector3.hpp"
+
+namespace BBM
+{
+	struct Utils
+	{
+		//! @brief Convert BBM Vector3f to RAY Vector3
+		static RAY::Vector3 toRAY(const BBM::Vector3f &wal);
+p
+		//! @brief default ctor
+		Utils() = default;
```

Comment:
Don't put constructors in a static class (a class where every member is static)

Reply:
This class can be deleted anyway

Reply:
nah i follow cooplean form

Reply:
even with useless stuff